presto-hive/pom.xml

```diff
@@ -351,7 +351,36 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
-    </dependencies>
+        
+        <!-- for coral -->
+        <dependency>
+            <groupId>com.linkedin.coral</groupId>
+            <artifactId>coral-presto</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.linkedin.calcite</groupId>
+            <artifactId>calcite-core</artifactId>
+            <version>1.21.1.1</version>
+            <classifier>shaded</classifier>
+            <scope>compile</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.linkedin.coral</groupId>
+            <artifactId>coral-hive</artifactId>
+            <exclusions>
```

Comment:
Remove exclusions since they are in the root POM

Reply:
I moved the exclusions to root POM.